packages/token-balance-cache/src/tbc.ts

```diff
@@ -53,6 +53,7 @@ export class TokenBalanceCache
 {
   private _nodes: { [id: number]: IChainNode } = {};
   private _providers: { [name: string]: BalanceProvider<any> } = {};
+  private _chainIds: { [id: string]: number } = {};
```

Comment:
Can you add a comment explaining what this object does? It's a little unintuitive / not clear from the declaration alone.

Reply:
thanks, good catch